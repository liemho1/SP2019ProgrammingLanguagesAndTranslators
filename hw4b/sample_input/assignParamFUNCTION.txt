{
    bojack = function(x) 
    {
      mrPB = function() { print("Mr. PB") };
      x = mrPB;
      cat(x)
    };
    print("neigh!")
}
