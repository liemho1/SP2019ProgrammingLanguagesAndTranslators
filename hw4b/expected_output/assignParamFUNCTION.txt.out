Line 5: Arg 1 must be integer
